hyco-https/tests/pipe.test.js

```diff
@@ -0,0 +1,138 @@
+var https = require('..');
+const Stream = require('stream');
+
+var ns = process.env.SB_HC_NAMESPACE ? process.env.SB_HC_NAMESPACE.replace(/^"(.*)"$/, '$1') : null;
+var path = process.env.SB_HC_PATH ? process.env.SB_HC_PATH : "a2";
+var keyrule = process.env.SB_HC_KEYRULE ? process.env.SB_HC_KEYRULE.replace(/^"(.*)"$/, '$1') : null;
+var key = process.env.SB_HC_KEY ? process.env.SB_HC_KEY.replace(/^"(.*)"$/, '$1') : null;
+
+expect(ns).toBeDefined();
+expect(path).toBeDefined();
+expect(keyrule).toBeDefined();
+expect(key).toBeDefined();
+
+var smallMessage = "SmallMessage";
+var kb = "";
+for (var i = 1024; i >= 0; i--) {
+    kb += String.fromCharCode(i % 128);
+}
+// let stream push 1kb at a time
+var over64kbMessage = []; 
+for (var j = 64; j >= 0; j--) {
+    over64kbMessage.push(kb);
+}
+
+jest.setTimeout(3000);
```

Comment:
>3000 [](start = 16, length = 4)

Does this mean the test must run in under 3 seconds? Would 5 or 10 allow the test to succeed in scenarios where this test is good but the machine is having issues (external to the tests)?

Reply:
That would be good idea

---
In reply to: [340272225](https://github.com/Azure/azure-relay-node/pull/44#discussion_r340272225) [](ancestors = 340272225)